agimus_controller_ros/agimus_controller_ros/agimus_controller.py

```diff
@@ -47,7 +47,12 @@ def __init__(self, node_name: str = "agimus_controller_node") -> None:
         self.param_listener = agimus_controller_params.ParamListener(self)
         self.params = self.param_listener.get_params()
         self.params.ocp.armature = np.array(self.params.ocp.armature)
-        self.traj_buffer = TrajectoryBuffer([(1, self.params.ocp.horizon_size - 1)])
+        self.params._trajectory_buffer_size_for_horizon = 0
```

Comment:
I have asked Theo not to do this but I guess he missed it or did not have time.

**It is a bad idea to set a private attribute of something else than `self`.**

Can you:
1. add `OCPParamsBaseCroco.total_time` that can be implemented as `sum(params.timesteps)`. Eventually, as Max suggested, you can make it a property computed in `__post_init__` and
2. replace `self.params._trajectory_buffer_size_for_horizon` by `self.ocp_params.total_time`

?